contracts/utils/Counters.sol

```diff
@@ -33,7 +33,7 @@ library Counters {
         uint256 value = counter._value;
         require(value > 0, "Counter: decrement overflow");
         unchecked {
-            counter._value = value - 1;
+            counter._value -= 1;
```

Comment:
You'd better to check [this](https://dev.to/javier123454321/solidity-gas-optimization-pt1-4271). Instead of directly using storage variable, it would take little gas if you use memory variables.